Improve examples - Correcting the URL to source code

brijendra98 opened the pull request:
The URL to the source code of the example that is being explained in the example is wrong. I am correcting it to match the needs of users.

pran1990 commented:
Hey @brijendra98 , thanks for the PR. Could you rebase against master?
https://stackoverflow.com/questions/8965781/update-an-outdated-branch-against-master-in-a-git-repo.

Let me know if you have issues.

brijendra98 commented:
I tried rebasing as you said but faced a lot of merge conflicts. 

```
CONFLICT (modify/delete): examples/upload/index.html deleted in 3f75cfa73e538ac6a0207de38eeb722138ff28b5 and modified in Spiff up examples. Version Spiff up examples of examples/upload/index.html left in tree.
CONFLICT (modify/delete): examples/team/index.html deleted in 3f75cfa73e538ac6a0207de38eeb722138ff28b5 and modified in Spiff up examples. Version Spiff up examples of examples/team/index.html left in tree.
CONFLICT (modify/delete): examples/team-as-user/index.html deleted in 3f75cfa73e538ac6a0207de38eeb722138ff28b5 and modified in Spiff up examples. Version Spiff up examples of examples/team-as-user/index.html left in tree.
CONFLICT (modify/delete): examples/styles.css deleted in 3f75cfa73e538ac6a0207de38eeb722138ff28b5 and modified in Spiff up examples. Version Spiff up examples of examples/styles.css left in tree.
Auto-merging examples/javascript/auth/index.html
CONFLICT (content): Merge conflict in examples/javascript/auth/index.html
CONFLICT (modify/delete): examples/index.html deleted in 3f75cfa73e538ac6a0207de38eeb722138ff28b5 and modified in Spiff up examples. Version Spiff up examples of examples/index.html left in tree.
CONFLICT (modify/delete): examples/download/index.html deleted in 3f75cfa73e538ac6a0207de38eeb722138ff28b5 and modified in Spiff up examples. Version Spiff up examples of examples/download/index.html left in tree.
CONFLICT (modify/delete): examples/basic/index.html deleted in 3f75cfa73e538ac6a0207de38eeb722138ff28b5 and modified in Spiff up examples. Version Spiff up examples of examples/basic/index.html left in tree.
error: Failed to merge in the changes.
Patch failed at 0001 Spiff up examples
The copy of the patch that failed is found in: .git/rebase-apply/patch

When you have resolved this problem, run "git rebase --continue".
If you prefer to skip this patch, run "git rebase --skip" instead.
To check out the original branch and stop rebasing, run "git rebase --abort".
```

pran1990 commented:
Hi @brijendra98 
Thanks for making this request. There are a few (fixable) issues, and I'm happy to help you walk through those.

1. You are still quite a few commits behind master. One possible reason for this is that you forked this repo long ago, and did not keep updating it.  One really fast way to fix this:
`cd /path/to/your/repo;
# for the step below you can use https url also, whatever you are comfortable with
git remote add dropbox_upstream git@github.com:dropbox/dropbox-sdk-js.git;
git fetch dropbox_upstream;
git reset --hard dropbox_upstream/master;
# Now make your changes
git push origin   master;
# Now go to github.com and update this pull request. (it'll likely happen by itself `

2.  You have update the URL to one example, can you do this for all the examples? That'll make things more uniform.

brijendra98 commented:
Hey @pran1990 ,
I'm really sorry for getting back so late. Had a busy week at work.

1. I forked this repository very recently and am 100% sure that everything in my fork is in sync with the original repo. The issue here is, I am trying to merge the "improve-examples" branch into the master branch and there are a lot of conflicts between these two branches. I am not sure how to resolve this issue.
One alternative that I can do is, making a pull request directly from the master branch in my fork of the repo to the master branch in the original repo.

2. Yes, I can definitely do that. Further I would also like to improve the UI of the examples. Please let me know about this so that I can proceed further.
